circle.yml

```diff
@@ -29,7 +29,10 @@ dependencies:
     - sudo apt-get install bc
     - pip install requests
     - curl https://sdk.cloud.google.com | bash
-    - bin/setup-circleci-secrets "$SECRET_PASSWORD"
+    - if [[ -n "$SECRET_PASSWORD" ]]; then
```

Comment:
Not a big fan of these tests everywhere - prs without the credentials will be filtered out by the ignore stanza at the top of the file.